ast/src/parsed/display.rs

```diff
@@ -190,15 +190,15 @@ impl Display for CallableRef {
     fn fmt(&self, f: &mut Formatter<'_>) -> Result {
         write!(
             f,
-            "{}{}.{} {}",
-            match self.params.inputs.len() {
-                0 => "".to_string(),
-                1 => format!("{} = ", self.params.inputs[0]),
-                _ => format!("({}) = ", self.params.inputs.iter().join(" ")),
+            "{}{}.{}({})",
+            match &self.params.outputs[..] {
```

Comment:
Is there a test that would have caught the input/output confusion?

Reply:
This wasn't caught before because `CallableRef` was not really being displayed: conversion happened later in the pipeline (i.e., `CallableRef` actually wasn't displayed anymore). Now it's part of the AST again and existing tests will catch it.